python/run_benchmark.sh

```diff
@@ -364,36 +364,39 @@ fi
 # Logistic Regression Classification
 if [[ "${MODE}" =~ "logistic_regression" ]] || [[ "${MODE}" == "all" ]]; then
 
-    data_path=${gen_data_root}/classification/r${num_rows}_c${num_cols}_float32_ncls${num_classes}.parquet
-
-    if [[ ! -d ${data_path} ]]; then
-        python $gen_data_script classification \
-            --n_informative $( expr $num_cols / 3 )  \
-            --n_redundant $( expr $num_cols / 3 ) \
-            --n_classes ${num_classes} \
-            --num_rows $num_rows \
-            --num_cols $num_cols \
-            --output_num_files $output_num_files \
-            --dtype "float32" \
-            --feature_type "array" \
-            --output_dir ${data_path} \
-            $common_confs
-    fi
-
-    echo "$sep algo: logistic regression $sep"
-    python ./benchmark/benchmark_runner.py logistic_regression \
-        --standardization False \
-        --maxIter 200 \
-        --tol 1e-30 \
-        --regParam 0.00001 \
-        --num_gpus $num_gpus \
-        --num_cpus $num_cpus \
-        --num_runs $num_runs \
-        --train_path ${data_path} \
-        --transform_path ${data_path} \
-        --report_path "report_logistic_regression_${cluster_type}.csv" \
-        $common_confs $spark_rapids_confs \
-        ${EXTRA_ARGS}
+    for num_classes in ${num_classes_list}; do
+
+        data_path=${gen_data_root}/classification/r${num_rows}_c${num_cols}_float32_ncls${num_classes}.parquet
```

Comment:
Can you change random forest classifier to reuse the same num_class=2 dataset?

Reply:
Done. The num_classes=2 dataset is now reused.